src/atomate2/vasp/jobs/adsorption.py

```diff
@@ -0,0 +1,437 @@
+"""Jobs used in the calculation of surface adsorption energy."""
+
+from __future__ import annotations
+
+import logging
+from collections import defaultdict
+from dataclasses import dataclass, field
+from typing import TYPE_CHECKING, Any
+
+from jobflow import Flow, Response, job
+from pymatgen.analysis.adsorption import AdsorbateSiteFinder
+from pymatgen.core import Element, Molecule, Structure
+from pymatgen.core.surface import SlabGenerator
+from pymatgen.io.vasp import Kpoints
+from pymatgen.io.vasp.sets import MPRelaxSet
+
+from atomate2.vasp.jobs.base import BaseVaspMaker
+
+# from atomate2.vasp.sets.core import RelaxSetGenerator, StaticSetGenerator
+
+if TYPE_CHECKING:
+    from atomate2.vasp.sets.base import VaspInputGenerator
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class BulkRelaxMaker(BaseVaspMaker):
+    """Maker for molecule relaxation.
+
+    Parameters
+    ----------
+    name: str
+        The name of the flow produced by the maker.
+    input_set_generator: VaspInputGenerator
+        The input set generator for the relaxation calculation.
+    """
+
+    name: str = "bulk_relax_maker__"
+    input_set_generator: VaspInputGenerator = field(
+        default_factory=lambda: MPRelaxSet(
+            user_kpoints_settings={"reciprocal_density": 200},
```

Comment:
This is quite dense. The normal density for relaxations is 60 and for statics 100. I imagine this could be expensive for surface calculations. Any reason for going this dense?